src/intl/en.json

```diff
@@ -1,16 +1,16 @@
 {
   "animation": {
-    "play": "Click to say Hello!",
+    "play": "Click to say 'Hello!'",
     "replay": "Replay?",
-    "hiSpan": "Hi!",
-    "mySpan": "my",
-    "nameSpan": "name",
-    "isSpan": "is",
-    "karolSpan": "Karol",
-    "hiMySpan": "Hi! My",
-    "hiMyNameSpan": "Hi! My name",
-    "hiMyNameIsSpan": "Hi! My name is",
-    "hiMyNameIsKarolSpan": "Hi! My name is Karol!"
+    "firstParagraph": "Hi!",
```

Comment:
and also here :P 
I think that you should read a book `Clean Code` written by Robert C. Martin. 
This is a very bad practice to add html tags to variable names

Reply:
thank god for ctrl+h